contrib/drivers/huawei/oceanstor/oceanstor.go

```diff
@@ -739,19 +739,26 @@ func (d *Driver) deleteZoneAndRemoveFCInitiators(wwns []string, hostId, hostGrpI
 func (d *Driver) getMappedInfo(hostName string) (string, string, string, string, error) {
 	hostId, err := d.client.GetHostIdByName(hostName)
 	if err != nil {
+		if IsNotFoundError(err) {
+			log.Warningf("host(%s) has been removed already, ignore it.", hostName)
+			return "", "", "", "", nil
+		}
+
 		return "", "", "", "", err
 	}
 
 	lunGrpId, err := d.client.FindLunGroup(PrefixLunGroup + hostId)
-	if err != nil {
+	if err != nil && !IsNotFoundError(err) {
```

Comment:
Trying to understand, why more strict check for isNotFoundError? If there is any other error than IsNotFoundError, we can proceed?

Reply:
this logic is, only if this error is NOT FOUND, we can proceed, otherwise we return err as before.